### PR TITLE
feat(properties-panel): migrate to properties panel 1.x

### LINKED ISSUE #145: Update `/properties-panel`
* [ ] Update `properties-panel` example to showcase `bpmn-js-properties-panel@1`
* [ ] Add link in the readme to the old version of the example

### PR DESCRIPTION
**Pull request by barmac**
Closes #145

### CODE REVIEW

File: properties-panel/README.md [hunk 1]
```diff
@@ -7,47 +7,103 @@ This example uses [bpmn-js](https://github.com/bpmn-io/bpmn-js) and [bpmn-js-pro
 
 This example is a node-style web application that builds a user interface around the bpmn-js BPMN 2.0 modeler.
 
-![demo application screenshot](https://raw.githubusercontent.com/bpmn-io/bpmn-js-examples/master/properties-panel/docs/screenshot.png "Screenshot of the modeler + properties panel example")
+![demo application screenshot](./docs/screenshot.png "Screenshot of the modeler + properties panel example")
```
Comment (nikku) [inline properties-panel/README.md:10]: I'd give the properties panel in the screenshot slightly more width + use a less prominent example (i.e. without colors).

It does make sense to use a real-world example though.

Comment (barmac) [reply properties-panel/README.md:10]: If you have a diagram I could use, feel free to share it! :)

Comment (barmac) [reply properties-panel/README.md:10]: I used the pizza collaboration and increased the panel width to 400px. @nikku please check now.

Comment (nikku) [reply properties-panel/README.md:10]: Great. If you move the diagram in sight (avoid participant-screen border collapsing) and focus an activity with less noise around it you'll end up with the perfect screenshot.

Comment (barmac) [reply properties-panel/README.md:10]: Done.

**Review by MaxTru — APPROVED** [review]
Great

**Pull request merged**